packages/office-ui-fabric-react/src/components/Button/SplitButton/SplitButton.styles.ts

```diff
@@ -32,18 +49,20 @@ export const getStyles = memoizeFunction((
       position: 'absolute',
     },
 
-    splitButtonMenuButton: {
+    splitButtonMenuButton: mergeStyleSets(splitButtonCommonStyles.splitButtonMenuButton, {
```

Comment:
If you're merging a single Style, you can use mergeStyles instead of mergeStyleSets.